src/utils/jsonTransform.test.js

```diff
@@ -0,0 +1,315 @@
+import {to_registration_service_format} from './jsonTransform';
+
+const testData = {
```

Comment:
Here we can import the test data file, just like we import the formatting method.